wdk-ddi-src/content/vpci/ns-vpci-vpci_interface_standard.md

```diff
@@ -0,0 +1,106 @@
+---
+UID: NS:vpci._VPCI_INTERFACE_STANDARD
+tech.root: pci
+title: VPCI_INTERFACE_STANDARD
+ms.date: 09/27/2022
+targetos: Windows
+description: Learn more about the VPCI_INTERFACE_STANDARD structure
+prerelease: false
+req.construct-type: structure
+req.ddi-compliance: 
+req.dll: 
+req.header: vpci.h
+req.include-header: 
+req.kmdf-ver: 
+req.lib: 
+req.max-support: 
+req.redist: 
+req.target-min-winverclnt: 
+req.target-min-winversvr: Windows Server 2012
+req.target-type: 
+req.typenames: VPCI_INTERFACE_STANDARD, *PVPCI_INTERFACE_STANDARD
+req.umdf-ver: 
+req.unicode-ansi: 
+topic_type:
+ - apiref
+api_type:
+ - HeaderDef
+api_location:
+ - vpci.h
+api_name:
+ - _VPCI_INTERFACE_STANDARD
+ - PVPCI_INTERFACE_STANDARD
+ - VPCI_INTERFACE_STANDARD
+f1_keywords:
+ - _VPCI_INTERFACE_STANDARD
+ - vpci/_VPCI_INTERFACE_STANDARD
+ - PVPCI_INTERFACE_STANDARD
+ - vpci/PVPCI_INTERFACE_STANDARD
+ - VPCI_INTERFACE_STANDARD
+ - vpci/VPCI_INTERFACE_STANDARD
+dev_langs:
+ - c++
+helpviewer_keywords:
+ - _VPCI_INTERFACE_STANDARD
+---
+
+## -description
+
+The **VPCI\_INTERFACE\_STANDARD** interface structure enables device drivers to access blocks of configuration data that is specific to a PCI Express (PCIe) virtual function (VF) of devices that support the single root I/O virtualization (SR-IOV) interface.
+
+This structure describes the **GUID\_VPCI\_INTERFACE\_STANDARD** interface.
+
+> [!NOTE]
+> The location of the VF configuration block and the format of the configuration data are defined by the independent hardware vendor (IHV) of the device. They are used only by the drivers of the PCIe physical function (PF) and the VF.
+
+## -struct-fields
+
+### -field Size
+
+The size, in bytes, of this structure.
+
+### -field Version
+
+The driver-defined interface version.
+
+### -field Context
+
+A pointer to interface-specific context information.
+
+### -field InterfaceReference
+
+A pointer to an [*InterfaceReference*](/windows-hardware/drivers/ddi/wdm/ns-wdm-_interface) routine that increments the interface's reference count.
```

Comment:
Change to root-relative path. We use ".." instead of ~, ie:
"[*InterfaceReference*](../wdm/ns-wdm_interface.md)"